lib/jormungandr/test/integration/Test/Integration/Jormungandr/Scenario/API/Transactions.hs

```diff
@@ -184,7 +196,45 @@ spec = do
             , expectResponseCode HTTP.status202
             ]
 
+        let wronglyEncodedTx = encodeWronglyTx (tx, wits) MsgTypeTransaction
+        let payload1 = Json [json|{
+                "payload": #{wronglyEncodedTx}
+            }|]
+        r1 <- request @ApiTxId ctx postExternalTxEp Default payload1
+        verify r1
+            [ expectErrorMessage errMsg404WronglyEncodedTxPayload
+            , expectResponseCode HTTP.status404
+            ]
+
     it "TRANS_EXTERNAL_CREATE_02 - proper single output transaction and \
+       \proper binary format - octet-stream" $ \ctx -> do
+
+        let toSend = 1 :: Natural
+
+        (tx, wits) <- fixtureExternalTx ctx toSend
+        let encodedSignedTx =
+                BL.fromStrict $ T.encodeUtf8 $
+                encodeTx (tx, wits) MsgTypeTransaction
+        let headers = Headers [ ("Content-Type", "application/octet-stream") ]
+        r <-
+            request @ApiTxId ctx postExternalTxEp headers (NonJson encodedSignedTx)
```

Comment:
The line above is **too long** according to our coding standards.

Recommend:
```hs
        r <- request
            @ApiTxId ctx postExternalTxEp headers (NonJson encodedSignedTx)
```

Reply:
corrected